lms/djangoapps/support/static/support/jsx/entitlements/components/Table/Table.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+import { Button, Hyperlink, Table } from '@edx/paragon';
+
+const entitlementColumns = [
+  {
+    label: 'User',
+    key: 'user',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Course uuid',
+    key: 'course_uuid',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Enrollment',
+    key: 'enrollment_course_run',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Mode',
+    key: 'mode',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Expired At',
+    key: 'expired_at',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Created',
+    key: 'created',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Modified',
+    key: 'modified',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Order',
+    key: 'order_number',
+    columnSortable: true,
+    onSort: () => {},
+  },
+  {
+    label: 'Actions',
+    key: 'button',
+    columnSortable: false,
+    hideHeader: false,
+    onSort: () => {},
+  },
+];
+
+const EntitlementTable = (props) => {
+  const entitlementData = props.entitlements.map((entitlement) => 
+    Object.assign({}, entitlement, {
+      expired_at: entitlement.expired_at ? moment(entitlement.expired_at).format('MMM. D, YYYY') : '',
+      created: moment(entitlement.created).format('MMM. D, YYYY'),
+      modified: moment(entitlement.modified).format('MMM. D, YYYY'),
+      order_number: <Hyperlink
+        destination={props.ecommerceUrl + entitlement.order_number + '/'}
+        content={entitlement.order_number || ''}
+      />,
+      button: <div> No Actions Currently Available </div>
+    })
+  );
+  return (
+    <div>
+      <Table
+        data={entitlementData}
+        columns={entitlementColumns}
```

Comment:
Do you need to add the `tableSortable` prop?

From [the Table README](https://github.com/edx/paragon/blob/master/src/Table/README.md#tablesortable-boolean-optional)

> Specifies whether the table is sortable. This setting supercedes column-level sortability, so if it is false, no sortable components will be rendered.

Looks like [the default prop value is `false`](https://github.com/edx/paragon/blob/master/src/Table/index.jsx#L177).